docstring_parser/tests/test_epydoc.py

```diff
@@ -403,3 +403,262 @@ def test_broken_meta() -> None:
+
+            @meta1: asd
+                1
+                    2
+                3
+            @meta2: herp
+            @meta3: derp
+            """,
+            "Short description\n\n@meta1: asd\n    1\n        2\n    3\n@meta2: herp\n@meta3: derp",
+        ),
+        (
+            """
+            Short description
+
+            @meta ene due rabe: asd
+            """,
+            "Short description\n\n@meta ene due rabe: asd",
+        ),
+        (
+            """
+            Short description
+
+            @param name: description 1
+            @param priority: description 2
+            @type priority: int
+            @param sender: description 3
+            @type sender: str? 
+            @type message: str?
+            @param message: description 4, defaults to 'hello'
+            @type multiline: str?
+            @param multiline: long description 5,
+                defaults to 'bye'
+            """,
+            "Short description\n\n@param name: description 1\n"
+            "@type priority: int\n"
+            "@param priority: description 2\n"
+            "@type sender: str?\n"
+            "@param sender: description 3\n"
+            "@type message: str?\n"
+            "@param message: description 4, defaults to 'hello'\n"
+            "@type multiline: str?\n"
+            "@param multiline: long description 5,\n"
+            "    defaults to 'bye'",
+        ),
+        (
+            """
+            Short description
+            @raise: description
+            """,
+            "Short description\n@raise: description",
+        ),
+        (
+            """
+            Short description
+            @raise ValueError: description
+            """,
+            "Short description\n@raise ValueError: description",
+        ),
+    ],
+)
+def test_compose(source: str, expected: str) -> None:
+    assert compose(parse(source)) == expected
+
+
+@pytest.mark.parametrize(
+    "source, expected",
+    [
+        (
+            """
+            Short description
+
+            @param name: description 1
+            @param priority: description 2
+            @type priority: int
+            @param sender: description 3
+            @type sender: str? 
+            @type message: str?
+            @param message: description 4, defaults to 'hello'
+            @type multiline: str?
+            @param multiline: long description 5,
+                defaults to 'bye'
+            """,
+            "Short description\n\n@param name:\n    description 1\n"
+            "@type priority: int\n"
+            "@param priority:\n    description 2\n"
+            "@type sender: str?\n"
+            "@param sender:\n    description 3\n"
+            "@type message: str?\n"
+            "@param message:\n    description 4, defaults to 'hello'\n"
+            "@type multiline: str?\n"
+            "@param multiline:\n"
+            "    long description 5,\n"
+            "    defaults to 'bye'",
```

Comment:
Let's be consistent about usage of `\n` and `"""` (here and in other tests)

Reply:
Could you clarify what you mean?

Reply:
The latter is different from the former because of the initial indent. For `source` these would test a slightly different thing than the tests as written, for `expected` writing tests in this way would require them to be dedented -- unless I'm misunderstanding something.

Reply:
That's true, but we can always reduce the indentation like so:
```python3
    """line 1
line 2
line 3
"""
```
and I think that's OK, but come to think of it I'm OK with your way too. Whichever you prefer

Reply:
ok - I went with making how "\n" appears in the docstrings more consistent. For oneliners I kept things how they are. For multiliners I kept "\n", but made sure that if \n appears in the string then they appear at the end.